app/models/equipment_item.rb

```diff
@@ -106,6 +106,15 @@ def update(current_user, new_params) # rubocop:disable all
     self
   end
 
+  def add_notes(current_user, contents)
+    return self if contents.empty?
+    new_notes = "#### #{current_user.md_link} made a note on "\
+      "#{Time.current.to_s(:long)}:\n\n#{contents[0]}"
```

Comment:
Let's get rid of the colon (`:`) at the end of the line since it looks weird coming after a time.